src/app/core/services/logger.service.ts

```diff
@@ -0,0 +1,159 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoggerService {
+  private sessionId = 'sample-session-id'; // Replace with actual session ID logic
```

Comment:
Add a method here to  generate a session Id too.

Reply:
added a method for generating session id